db/migrate/20180810055852_add_secret_generated_at_to_applications.rb

```diff
@@ -0,0 +1,5 @@
+class AddSecretGeneratedAtToApplications < ActiveRecord::Migration[5.2]
+  def change
+    add_column :oauth_applications, :secret_generated_at, :datetime, default: -> { 'CURRENT_TIMESTAMP' }, null: false
```

Comment:
This is going to have the default value be the current time when the migration is run, showing all old client secrets again for another 24 hours.  Just want to make sure you are aware of this.